hwdb.d/60-keyboard.hwdb

```diff
@@ -1009,7 +1009,7 @@ evdev:name:ThinkPad Extra Buttons:dmi:bvn*:bvr*:bd*:svnLENOVO*:pn*:*
  KEYBOARD_KEY_15=volumedown
  KEYBOARD_KEY_16=mute
  KEYBOARD_KEY_17=prog1
- KEYBOARD_KEY_1a=f20                                    # Microphone mute button; should be micmute
+ KEYBOARD_KEY_1a=micmute                                # Microphone mute button
```

Comment:
This has been the default kernel mapping since at least 2011, so I guess this could be dropped.

Note the same can be said for some of the other mappings of this driver quite  few just repeat the default kernel mappings (but not all ...)